test/e2e/Tests/Constants.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E
+{
+    internal class Constants
+    {
+        public static IConfiguration Configuration = TestUtility.GetTestConfiguration();
+
+        internal static string FunctionsHostUrl = Configuration["FunctionAppUrl"] ?? "http://localhost:7071";
```

Comment:
nit: since these are in a "Constants" file, we should probably make them readonly.

```suggestion
        internal static readonly string FunctionsHostUrl = Configuration["FunctionAppUrl"] ?? "http://localhost:7071";
```